api/src/main/java/com/messagebird/MessageBirdClient.java

```diff
@@ -1587,4 +1569,47 @@ private void verifyOffsetAndLimit(Integer offset, Integer limit) {
             throw new IllegalArgumentException("Limit must be > 0");
         }
     }
-}
\ No newline at end of file
+
```

Comment:
Can you add note related methods as others have?
/**
 * Function to delete a webhook
 *
 * @param id A unique random ID which is created on the MessageBird platform
 * @throws NotFoundException     if id is not found
 * @throws GeneralException      general exception
 * @throws UnauthorizedException if client is unauthorized
 */